packages/flutter/lib/src/widgets/focus_traversal.dart

```diff
@@ -1570,10 +1570,41 @@ class ReadingOrderTraversalPolicy extends FocusTraversalPolicy
   /// {@macro flutter.widgets.FocusTraversalPolicy.requestFocusCallback}
   ReadingOrderTraversalPolicy({super.requestFocusCallback});
 
+  /// Sorts the input focus nodes into reading order.
+  static Iterable<FocusNode> sort(Iterable<FocusNode> nodes) {
```

Comment:
Nit: Is there any use in adding a non-widget `test` for this directly?

Reply:
the current test coverage should be good enough